aspnetcore/blazor/security/server/index.md

```diff
@@ -173,6 +173,25 @@ For more information on the Blazor Identity UI and guidance on integrating exter
 
 [!INCLUDE[](~/includes/aspnetcore-repo-ref-source-links.md)]
 
+## Manage authentication state in Blazor Web Apps
+
+*This section applies to Blazor Web Apps that adopt:*
+
+* *Interactive server-side rendering (interactive SSR) and CSR.*
+* *Client-side rendering (CSR).*
```

Comment:
Are these acronyms used elsewhere? I haven't seen them before.

Reply:
Yes, right from the start ...

https://learn.microsoft.com/en-us/aspnet/core/blazor/components/render-modes?view=aspnetcore-8.0

Set up in consultation with Dan. Based on his remarks about industry use, "client-side rendering"/"CSR" doesn't get the "interactive" word applied, and there's an dagger remark there under the table to call that out :point_up:.

I always spell these out (static SSR/interactive SSR) first in a section (e.g., "interactive server-side rendering (interactive SSR)"), and then I use "interactive SSR" throughout the section. I figured that "SSR" might be too cryptic for many devs and that it always should be spelled out initially before using the acronym.